### PR TITLE
Installed Swashbuckle and MediatR packages

### LINKED ISSUE #15: Add Swashbuckle and MediatR

### PR DESCRIPTION
**Pull request by laska-sl**

### CODE REVIEW

File: ArcadiaParties-SPA/angular.json [hunk 1]
```diff
@@ -119,6 +119,10 @@
...
         }
       }
-    }},
-  "defaultProject": "ArcadiaParties-SPA"
-}
+    }
+  },
+  "defaultProject": "ArcadiaParties-SPA",
+  "cli": {
+    "analytics": false
```
Comment (sponkrashin) [inline ArcadiaParties-SPA/angular.json:126]: What is this for?

File: ArcadiaParties.API/Properties/launchSettings.json [hunk 2]
```diff
@@ -21,10 +21,10 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "weatherforecast",
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "applicationUrl": "http://localhost:5000",
```
Comment (sponkrashin) [inline ArcadiaParties.API/Properties/launchSettings.json:24]: Removed URL will be used if you enable https in application startup. It is not needed to remove this.

File: ArcadiaParties.API/Startup.cs [hunk 3]
```diff
@@ -26,6 +26,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
+
```
Comment (sponkrashin) [inline ArcadiaParties.API/Startup.cs:25]: Accidentally added empty line

File: ArcadiaParties.API/Startup.cs [hunk 4]
```diff
@@ -36,8 +37,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseHttpsRedirection();
```
Comment (sponkrashin) [inline ArcadiaParties.API/Startup.cs:39]: Why?

**Review by sponkrashin — CHANGES_REQUESTED** [review]

**Comment by sponkrashin** [comment]
You should not only add Swagger, but also make all preparations to use it. I mean, services configuration, requests pipeline configuration.

**Comment by sponkrashin** [comment]
Also you can add a project where all queries and commands handlers needed for MediatR will be created, and register them according to documentation. You don't need to create handlers themselves, only an empty project. 
https://github.com/jbogard/MediatR/wiki#aspnet-core

File: ArcadiaParties-SPA/angular.json (hunk 1 above)
Comment (laska-sl) [reply ArcadiaParties-SPA/angular.json:126]: Was added after first "ng serve"

File: ArcadiaParties.API/Startup.cs (hunk 4 above)
Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs:39]: I see this when navigate to http://...
image

File: ArcadiaParties-SPA/angular.json (hunk 1 above)
Comment (sponkrashin) [reply ArcadiaParties-SPA/angular.json:126]: OK.

File: ArcadiaParties.API/Startup.cs (hunk 4 above)
Comment (sponkrashin) [reply ArcadiaParties.API/Startup.cs:39]: OK, let's remove it.

File: ArcadiaParties.API/Properties/launchSettings.json (hunk 2 above)
Comment (sponkrashin) [reply ArcadiaParties.API/Properties/launchSettings.json:24]: Keep as is

File: ArcadiaParties.API/Startup.cs (hunk 3 above)
Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs]: Solved in "Connect temp db to" PR

Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs:25]: Solved

**Comment by sponkrashin** [comment]
Does swagger work? It should show an empty swagger page.

**Comment by laska-sl** [comment]
> Does swagger work? It should show an empty swagger page.

Yeah, it works
image

**Comment by sponkrashin** [comment]
Great. What about MediatR? Could you add an empty project for its handlers and register all classes from it as it is described in the documentation?

**Pull request merged**